### PR TITLE
Fix stylesheet bugs

### PR DESCRIPTION
**Pull request by NemTechCN**
task #264;
task #262;
task #233;

### CODE REVIEW

File: src/core/utils/mosaics.ts [hunk 1]
```diff
@@ -9,17 +9,51 @@ import {AppMosaic} from '@/core/model'
...
+    return items
+}
+
+export const renderMosaics = (
+    mosaics: Mosaic[],
+    mosaicList: AppMosaic[],
+    currentXem: string): any => {
     const items = mosaics
-        .map((mosaic) => {
+        .map((mosaic: any) => {
```
Comment (decentraliser) [inline src/core/utils/mosaics.ts]: mosaic: AppMosaic

Comment (decentraliser) [reply src/core/utils/mosaics.ts]: If you have to put `any` to make it work, there is a problem

File: src/views/monitor/monitor-transaction-list/TransactionList.vue [hunk 2]
```diff
@@ -63,7 +63,13 @@
                         :class="['overflow_ellipsis',!c.isReceipt ? 'green' : 'red']"
                         v-if="c.rawTx.type === TransactionType.TRANSFER"
                 >
+
                     {{ renderMosaics(c.rawTx.mosaics, mosaicList, currentXem) }}
+                  <!--                   <span v-for="m in renderMosaics(c.rawTx.mosaics, mosaicList, currentXem) ">-->
```
Comment (decentraliser) [inline src/views/monitor/monitor-transaction-list/TransactionList.vue]: remove comments if useless

File: src/core/utils/mosaics.ts (hunk 1 above)
Comment (NemTechCN) [reply src/core/utils/mosaics.ts]: fixed

File: src/views/monitor/monitor-transaction-list/TransactionList.vue (hunk 2 above)
Comment (NemTechCN) [reply src/views/monitor/monitor-transaction-list/TransactionList.vue]: fixed

**Pull request merged**